src/layers/core/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -127,9 +132,17 @@ export default class SolidPolygonLayer extends Layer {
     const geometryConfigChanged = props.extruded !== oldProps.extruded ||
       props.wireframe !== oldProps.wireframe || props.fp64 !== oldProps.fp64;
 
-     // When the geometry config  or the data is changed,
-     // tessellator needs to be invoked
-    if (changeFlags.dataChanged || geometryConfigChanged) {
+    // check if updateTriggers.getElevation has been triggered
+    const getElevationTriggered = changeFlags.updateTriggersChanged &&
```

Comment:
How does this work? `changeFlags.updateTriggersChanged` was introduced as part of a major lifecycle refactor in 4.2 alpha. It doesn't (shouldn't) exist in 4.1?